GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -495,8 +562,8 @@ static void StraightenLanes(int startIndex, int lastStraightenIndex, int lastLoo
                 {
                     goBackLimit = Math.Max(goBackLimit, currentIndex - _straightenLanesLookAhead);
                     bool moved = false;
-                    IRevisionGraphRow currentRow = localOrderedRowCache[currentIndex];
-                    IRevisionGraphRow previousRow = localOrderedRowCache[currentIndex - 1];
+                    RevisionGraphRow currentRow = localOrderedRowCache[currentIndex];
+                    RevisionGraphRow previousRow = localOrderedRowCache[currentIndex - 1];
```

Comment:
why?

Reply:
No virtual call
Analyzer suggestion to use specific types if possible